README.md

```diff
@@ -1,29 +1,45 @@
-# Forest
+# 🌲 Forest 
 ![](https://github.com/ChainSafe/forest/workflows/Rust%20CI/badge.svg?branch=master)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![](https://img.shields.io/twitter/follow/espadrine.svg?label=Follow&style=social)](https://twitter.com/chainsafeth)
 
-### [WIP]
-A Filecoin implementation written in Rust.
 
-Based on the [Filecoin Protocol Specification](https://filecoin-project.github.io/specs/).
+Forest is an implementation of [Filecoin](https://filecoin.io/) written in Rust. The implementation will take a modular approach to building a full Filecoin node in two parts — (i) building Filecoin’s security critical systems in Rust from the [Filecoin Protocol Specification](https://filecoin-project.github.io/specs/), specifically the virtual machine, blockchain, and node system, and (ii) integrating functional components for storage mining and storage & retrieval markets to compose a fully functional Filecoin node implementation.
 
-## Development
+❗**Current development should be considered a work in progress.**
 
-See [`Makefile`](./Makefile) for additional commands.
+Our crates:
 
-#### Build/Run
-`cargo build && ./target/debug/node`
+| crate | description |
+|-|-|
+| `blockchain` | chain structures and syncing functionality |
+| `crypto` | verification and signature definitions |
+| `encoding` | used for encoding and decoding |
+| `ipld` | IPLD data model for content-addressable data |
+| `node` | networking synchronization and storage |
+| `vm` | state transition and actor, message definitions |
```

Comment:
Yeah, I just figured this would act as a summary more or less rather than an exact representation but I can include the full crate names if that's what popular opinion is. Specifically, `forest_ipld`, and  `forest_encoding`

Reply:
Doesn't really matter, fine the way it is